src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -113,10 +115,11 @@ export const NewAnnotationForm = ({
 		);
 
 		areUiErrors.current = false;
+		validateModFormFields(newAnnotation, modFormFields, uiErrorMessages, setUiErrorMessages, areUiErrors);
 		validateFormBioEntityFields(newAnnotation, uiErrorMessages, setUiErrorMessages, areUiErrors);
 		if (areUiErrors.current) {
-			newAnnotationDispatch({type: "UPDATE_ERROR_MESSAGES", errorType: "errorMessages", errorMessages: uiErrorMessages});
-			newAnnotationDispatch({ type: "SET_IS_ENABLED", value: false });
+			//newAnnotationDispatch({type: "UPDATE_ERROR_MESSAGES", errorType: "errorMessages", errorMessages: uiErrorMessages});
+			newAnnotationDispatch({ type: "SET_IS_ENABLED", value: true });
 
```

Comment:
Also, want changing this boolean break the logic for validateBioEntityFields? If there are uiErrors with the bioEntityFields this should not be enabled.